lib/kubeclient/common.rb

```diff
@@ -24,6 +24,10 @@ def initialize_client(
         password: nil,
         bearer_token: nil,
         bearer_token_file: nil
+      },
+      socket_options: {
```

Comment:
I am probably missing something here, but why is there a need to do the socket_options hash initialization both here and in kubeclient.rb line 55? the code looks the same and kubeclient.rb calls common.rb anyway